managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+8. Click **Save**. Upon saving, the repository is imported and scanned for any included workflows.
+
+   ![Repository details](../images/embedworkflow_addnewrepository_details.png){: caption="Repository details" caption-side="bottom"}
+
+9. Confirm that the synchronization with your repository is successful by viewing the list of workflows that are detected for the repository.
+
+**Note:** If you are adding a repository that is not publicly available, add an SCM (source control) credential for accessing the repository. This credential is separate from the workflow credentials that you configure in the procedure for adding the repository. Source Control credentials have several attributes that can be configured:
+
+   - **Username**: The username to use in conjunction with the source control system.
+   - **Password**: The password to use in conjunction with the source control system.
+   - **Private key passphrase**: If the SSH private key used is protected by a passphrase, you can configure a key passphrase for the private key.
+   - **Private Key**: Copy or drag and drop the actual SSH Private Key to be used to authenticate the user to the source control system by using SSH.
+
+### Viewing the List of Workflows in a Repository
+
+When your repository is added, the repository shows in the *Repositories* list. You can now click the entry for the repository to open it so that you can scan for existing workflows.
+If multiple workflows are found, click the corresponding number to view the list of workflows, **3** in this case.
+
+   ![Repository workflows](../images/embedworkflow_workflownumbers.png){: caption="Repository workflows" caption-side="bottom"}
+
+Alternatively, click **Automation** > **Embedded Workflows**. Then, click **Workflows** to see the workflows list.
+
+### Refreshing Repositories
+
+{{ site.data.product.title_short }} allows you to refresh a targeted repository or all repositories in your inventory to ensure that your workflows are current.
+
+Refresh a targeted repository:
+
+1.  Browse to the menu: **Automation** > **Embedded Workflows** > **Repositories**.
+
+2.  Click a repository.
+
+3.  Click **Configuration**, then ![Refresh this Repository](../images/2003.png) (**Refresh this Repository**).
+
+Alternately, you can refresh some or all repositories from the list view:
+
+1.  Browse to the menu: **Automation** > **Embedded Workflows** > **Repositories**.
+
+2.  Check those repositories to refresh. Click **Check All** to select all repositories.
+
+3.  Click **Configuration**, then ![Refresh Selected Workflow Repositories](../images/2003.png)(**Refresh Selected Workflow Repositories**).
+
+### Adding Credentials
+
+Credentials are used by {{ site.data.product.title_short }} for authentication when running embedded workflows against machines, synchronizing with inventory sources, and importing project content from a version control system.
```

Comment:
Is this a duplicate of https://github.com/ManageIQ/manageiq-documentation/pull/1752/files#diff-a944ad335565b3b6f0c1776d24ce585d3b42531e2d45bc20a5eaab1950a871e4R43 ?

If we need to have both then the same comment applies

Reply:
I have made the same change in this section.